scripts/restoreDump.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env babel-node
+
+import fs from 'fs'
+import { execSync } from 'child_process'
+import { Log, env, cli } from 'decentraland-commons'
+
+import { loadEnv, resolvePath, runpsql } from './utils'
+
+const log = new Log('restoreDump')
+
+export async function restoreDump(dumpPath) {
+  if (!dumpPath) {
+    throw new Error(
+      'Empty dump path. Please add it as an argument when running this script'
+    )
+  }
+
+  const connectionString = env.get('CONNECTION_STRING')
+  if (!connectionString) {
+    throw new Error('Connection string empty. Please fill your env file')
+  }
+
+  const shouldContinue = await cli.confirm(
+    `Careful! this will DROP your database and re CREATE it
+Do you wish to continue?`
+  )
+  if (!shouldContinue) return process.exit()
+
+  const databaseName = connectionString.split('/').pop()
+  const dumpFullPath = resolvePath(dumpPath)
+
+  if (!fs.existsSync(dumpFullPath)) {
+    throw new Error(`Couldn't find dump file "${dumpFullPath}"`)
+  }
+
+  log.info(`Reseting ${databaseName}`)
+  execSync(`psql -c 'DROP DATABASE IF EXISTS ${databaseName}'`)
```

Comment:
How are the user credentials and the host being managed?

Reply:
The script doesn't do anything regarding credentials, it's the responsibility of the entity that runs it (like the rest of the project).

You can use `~/.pgpass` or env variables like `PGUSER` and `PGPASSWORD`

Reply:
I think it's 🆗 I wasn't aware if it was a design decision or something you forgot to add.